SECURITY.md

```diff
@@ -10,6 +10,7 @@
 |                  |            |         |                  | v0.28.3                | 2024-07-08 | v0.28.3 |
 |                  |            |         |                  | v0.28.4                | 2025-02-10 | v0.28.4 |
 |                  |            |         |                  | v0.28.5                | 2025-02-21 | v0.28.5 |
+|                  |            |         |                  | v0.28.6                | 2025-08-27 | v0.28.6 |
```

Comment:
I'm tempted to delete all the older versions from this table. At this point, we're only doing bugfixes in 0.28.x, right?